bindings/stronghold-nodejs/build/Dockerfile.alpine

```diff
@@ -0,0 +1,5 @@
+FROM node:16-alpine
+
+# libc6-compat for gcc compatibility and binutils for debugging
+RUN apk --no-cache add libc6-compat binutils
+RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2
```

Comment:
Is this used within this project or only in the GitHub Actions workflow? Should we maybe move it to the `.github` directory?